backend/src/election/api.rs

```diff
@@ -312,7 +315,13 @@ pub async fn election_import(
     // Process polling stations
     let mut polling_places = None;
     let mut number_of_voters = 0;
+
     if let Some(polling_station_data) = edu.polling_station_data {
+        // If polling stations are submitted, file name must be also
```

Comment:
I would also add the same file name check to the validate endpoint, just to keep the logic synchronized between the two.